test/e2e/specs/fakerp/admin.go

```diff
@@ -103,4 +134,182 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]"
 		By("comparing the update blob before and after an update")
 		Expect(reflect.DeepEqual(before, after)).To(Equal(true))
 	})
+
+	It("should be able to configure azure file persistent volumes", func() {
+		const (
+			prefix = "azure-file"
+		)
+
+		var (
+			pvName string
+		)
+
+		pvName = fmt.Sprintf("%s-pv", prefix)
+		// need a way to lessen the chances of name collision in storage account names
+		// and also they must lowercase letters and numbers
+		clusterSHA := sha1.Sum([]byte(azCli.Config.ResourceGroup))
+		// storage account names are capped at 24 chars
+		accountName := fmt.Sprintf("azf%x", clusterSHA)[:24]
+
+		// create storage accounts and get their keys
+		By(fmt.Sprintf("Creating %s storage account", accountName))
+		_, err = azCli.Accounts.GetOrCreate(
+			ctx,
+			azCli.Config.ResourceGroup,
```

Comment:
use os.Getenv() etc. here for now - let's do a clear up of all the jobs in one go rather than having some do things one way and some another